src/frontends/tensorflow/src/input_model.hpp

```diff
@@ -16,11 +16,13 @@ namespace tensorflow {
 
 class OpPlace;
 class TensorPlace;
+class SMVariablesIndex;
```

Comment:
Let us have `SavedModelVariablesIndex`. It looks better:) and be more aligned with other class names for which we do not shorten names

Reply:
Better or aligned with code style for the repo? Why do we use OpPlace instead of OperationPlace then? If we have some rules about naming I would like to read them.

Reply:
`OpPlace` is a bug of course. I do not like it:)
Come on, it is easy fix just 10 min. to replace:)

Reply:
Question isn't about easy or not. If it isn't documented requirement - I can spend whole life for passing one code review because requirements might be changed daily. I don't want to play this game.